Readme.asciidoc

```diff
@@ -1,68 +1,116 @@
+- Shared Memory Service - In this image, JBoss Data Grid is configured as a shared memory service with fault-tolerance. That means any data stored by the service will be preserved across restarts. Fault-tolerance is achieved by replicating data across multiple nodes. The data is copied to a specific number of nodes in the cluster using the `owners` setting. This ensures that, in the case of failure, the grid will not lose data as long as the number of failed nodes is less than the number of copies (owners). This also works when nodes are shutdown or restarted manually.
 
 The services are based on https://www.redhat.com/en/technologies/jboss-middleware/data-grid[JBoss Data Grid] technology and they expose various endpoints and configurations depending on the use case.
 
-What's inside?
---------------
+== Getting started
 
-This repository has the following structure:
+Before you begin working with JDG Online Services, you will need a working OpenShift cluster as well as `oc` client in your `$PATH`. See https://openshift.io/[OpenShift documentation] how to achieve this.
 
-```
-+- Root
-    +-- caching-service - This service
-               +-- image.yaml - Image build recipe
-               +-- Makefile - A set of useful scripts for development work
-               +-- templates - OpenShift templates for this service
-               +-- modules - Modules used for the image
-               +-- functional-tests - Full end-to-end tests with Arquillian Cube
-```
+=== Installing JDG Online Services into Service Catalog
 
-The `image.yaml` uses Python framework called http://concreate.readthedocs.io/en/develop/[Concreate] to build an image. The easiest way to start the build is to invoke `concreate image.yaml target` and invoke a standard Docker build from `target` directory.
+JDG Online Services are supported on OpenShift Container Platform 3.7 (and equivalents such as OpenShift Online). In order to
+ make JDG Online Services available in the Service Catalog, the administrator needs to add all templates into `openshift`
+ namespace. This can be done by invoking `oc create -f templates/caching-service.json -n openshift` and
+ `oc create -f templates/shared-memory-service.json -n openshift` commands.
 
-The `Makefile` contains lots of small, useful scripts. In order to perform a full end-to-end test, invoke `make test-ci`. This will spin up a local OpenShift cluster, build the image, install the template, invoke end-to-end tests and kill the cluster.
 
-The `templates` directory contains a template for Service Catalog. The easiest way to install it is to use `make install-templates-in-openshift-namespace`.
+=== Instantiating JDG Online Service
 
-The `modules` directory contains modules used in `image.yaml`. See http://concreate.readthedocs.io/en/develop/[Concreate] documentation for more details.
+Once the above templates are correctly installed, one can instantiate them using proper icon in the catalog.
```

Comment:
I would say 'you' instead of 'one'. and 'appropriate' instead of 'proper'. Also add a 'the' in front of appropriate.

"Once the above templates are correctly installed, you can instantiate them using the appropriate icon in the catalog."

Reply:
Done.